### DISCUSSION TITLE
pldd run failed with Operation not permitted in github action

### DISCUSSION THREAD

**Discussion by yunnysunny** [community/community#84727] [Actions] [answered]
### Select Topic Area

Question

### Body

I run pldd command in github action, but it failed with Operation not permitted.
https://github.com/whyun-docker/node/blob/1f5505ff292d3fe5ec565ed01f9e30efe87842d3/.github/workflows/ubuntu.yml#L137-L141
The user run the runner is root. But it still have no enough permission.

**Comment by hejoefs** [comment]
Hey,

This is likely due to insufficient privileges to access the process ID (PID) namespace. Try to grant the runner the required privileges
For example, run the following command to grant the runner the ability to access the PID namespace:
`sudo chown -R runner:runner /proc`

Once you have granted the runner the required privileges, you should be able to run the `pldd` command successfully without encountering the 'Operation not permitted' error. 

Wish you the best!

**Reply by yunnysunny** [reply]
The user I ran the runner is root . And it doesn't have sudo command in the runner. I have modified the commands to show it.

**Comment by yunnysunny** [comment, marked as answer]
I found the reason now. Add `--privileged` parameter to  command `docker run` will resolve the problem.
